lib/datasource/adoptium-java/index.spec.ts

```diff
@@ -10,13 +10,6 @@ const jre = loadFixture('jre.json');
 function getPath(page: number, imageType = 'jdk'): string {
   return `/v3/info/release_versions?page_size=${pageSize}&image_type=${imageType}&project=jdk&release_type=ga&sort_method=DATE&sort_order=DESC&vendor=adoptium&page=${page}`;
 }
-function* range(start: number, end: number): Generator<number, number, number> {
```

Comment:
What's the issue about this function?

Reply:
Just wasn't sure if my fix would be right